code/src/layouts/page-loading-fallback/PageLoadingFallback.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { render, screen } from "@testing-library/react";
+import PageLoadingFallback from "@/layouts/page-loading-fallback/PageLoadingFallback";
+
+describe("Test PageLoadingFallback", () => {
+  test("Should render loading fallback", () => {
+    render(<PageLoadingFallback />);
```

Comment:
try to use `beforeEach` as I see this logic repeated for each test

Reply:
Thanks for suggestion, in one of those tests I pass different prop, which must not be passed in other tests, I guess `beforeEach` won't suit here. I think some render function that can be created, but I don't see much sense in it because there is not much difference between calling, for example `renderLoadingCallback()` and using `render(<PageLoadingFallback />)`

Reply:
> Thanks for suggestion, in one of those tests I pass different prop, which must not be passed in other tests, I guess `beforeEach` won't suit here. I think some render function that can be created, but I don't see much sense in it because there is not much difference between calling, for example `renderLoadingCallback()` and using `render(<PageLoadingFallback />)`

I agree. Creation separate fn to render it is not a good idea. 
I thought maybe to mock some props data and saw that you use only `fullScreen` prop and it will be some useless fn which render your props component handle it and call it for each test